lib/preface_specs/decidable.mli

```diff
@@ -0,0 +1,104 @@
+(** [Decidable] is a "Contravariant Alternative". *)
+
+(** {2 Laws}
+
+    To have a predictable behaviour, the instance of [Decidable] must obey some
+    laws.
+
+    + All {!module:Divisible} laws
+    + [choose Either.left x (lose f) = x]
+    + [choose Either.right x (lose f) = x]
```

Comment:
Change this law to `choose Either.right (lose f) x = x`

Reply:
Thanks!